BrainCloudClient/Assets/BrainCloud/LitJson/JsonData.cs

```diff
@@ -372,6 +372,13 @@ public JsonData (object obj)
                 return;
             }
 
+            if (obj is ulong)
```

Comment:
I'm not fond of modifying the third party directly... what if we update it?

Reply:
Ya.. i agree.. i've made a few other changes to their lib so that we can parse more things out of the box...